packages/xod-cli/src/xodc-transpile.js

```diff
@@ -8,6 +8,8 @@ import { loadProject, readJSON, writeFile } from 'xod-fs';
 import { transformProject, transpile } from 'xod-arduino';
 import * as msg from './messages';
 
+const bundledWorkspace = path.resolve(__dirname, 'workspace');
```

Comment:
Copying to `./bin/workspace/__lib__` looks strange. Why is the `workspace` medium required? Is it a binary (`bin` parent required)?

Maybe we can copy the libraries simply to `packages/xod-cli/__lib__`?